### PR TITLE
fix 1971/11/11 missing setlist #46

### PR DESCRIPTION
**Pull request by jacobcharleskillian**
fix 1971/11/11 missing setlist #46

### CODE REVIEW

File: data/gdshowsdb/1971.yaml [hunk 1]
```diff
@@ -1309,7 +1309,63 @@
...
+    - :uuid: 4abb209c-d53f-4cff-914b-3b494bbd42b7
+      :name: Tennessee Jed
+      :segued: false
+    - :uuid: 5727375f-3d50-479b-8dce-1c391fec3ef3
+      :name: Jack Straw
+      :segued: false
+    - :uuid: a44ab626-62a1-470d-89dd-e378e7ab2e41
+      :name: Casey Jones
+      :segued: false
+    - :uuid: a9766e2a-1949-4bc4-8317-2deb0c7625be
```
Comment (jefmsmit) [inline data/gdshowsdb/1971.yaml]: A new set should start here.

File: data/gdshowsdb/1971.yaml [hunk 2]
```diff
@@ -1309,7 +1309,63 @@
...
+    - :uuid: b6acc05d-4eea-4fb7-bfdd-6c1a3bd0762f
+      :name: One More Saturday Night
+      :segued: false
+    - :uuid: 8226ed8c-138e-4af5-b1e3-7c83c00f88cb
+      :name: Truckin'
+      :segued: false
+    - :uuid: 1ed53b41-4045-4895-9e96-1e3fce71dc6e
+      :name: Sugar Magnolia
+      :segued: false
+    - :uuid: a4b6c41a-8cb6-4c69-9835-f0e3821c3d3a
```
Comment (jefmsmit) [inline data/gdshowsdb/1971.yaml]: I don't include Sunshine Daydream as separate unless there is a split or a reprise.

File: data/gdshowsdb/1971.yaml [hunk 3]
```diff
@@ -1309,7 +1309,63 @@
...
+    - :uuid: 83a4262e-0e76-4b59-866e-a77de349929c
+      :name: Not Fade Away
+      :segued: true
+    - :uuid: f3f42473-ae97-407d-864d-0eade6b1711c
+      :name: Goin' Down The Road Feeling Bad
+      :segued: true
+    - :uuid: 83a4262e-0e76-4b59-866e-a77de349929c
+      :name: Not Fade Away
+      :segued: false
+    - :uuid: 2503ca6d-f21d-47e3-99f0-dd9f318baade
```
Comment (jefmsmit) [inline data/gdshowsdb/1971.yaml]: This is the encore and therefore should be in its own set.

File: data/gdshowsdb/1971.yaml (hunk 1 above)
Comment (jacobcharleskillian) [reply data/gdshowsdb/1971.yaml]: Yep.  Encores are sets.  Got it.  :)

**Comment by jefmsmit** [comment]
@jacobcharleskillian I don't really have any rules like that written down unfortunately. Up until now I was the only one maintaining the data set.

**Pull request merged**